contrib/doc/GafferUserGuide/tutorials/commandHistoryAndCompletionForCLI.txt

```diff
@@ -5,7 +5,7 @@
 Out of the box, the gaffer command line app "gaffer cli" isn't particularly useable - there's no tab-to-complete or history or line editing. This is because the python module responsible for such things isn't compatible with the BSD license, and therefore can't be distributed with Gaffer. This short how-to provides a workaround for that.
 
 ==== Enabling readline and command completion
-Chances are you already have a system install of python, and it has the readline module. If so, the following startup script will locate it and set it up, along with some nice tab-based completion. Just cut and paste the text into a ~/gaffer/startup/cli/completion.py file. 
+Chances are you already have a system install of python, and it has the readline module. If so, the following startup script will locate it and set it up, along with some nice tab-based completion. Just cut and paste the text into a ~/gaffer/<gafferCompatibilityVersion>/startup/cli/completion.py file.
```

Comment:
Apologies for butting in, but I noticed we're changing content inside `contrib/doc`. This shouldn't affect the docs build. I've been working under the assumption that this was all leftover material from pre-Sphinx. Is that incorrect?

Reply:
No, that is correct. I've been keeping contrib/doc around as a source to borrow from, but I think we're getting to the point where it has served its purpose. Feel free to make a PR to remove it if it's of no further use to you...

Reply:
Yeah, sorry, I just did a bulk search/replace in CLion for the docs, I didn't really look at the file paths... :+1: for deleting those contrib/docs altogether at this point.